### ISSUE TITLE
Is there supposed to be a map?

### ISSUE THREAD

**Issue by PurpleVsGreen** [OvertureMaps/io-site#171]
### Description

I could be mistaken, but I'd really expect some kind of map view or other data visuals.

### What browser are you seeing the problem on? What version are you running?

Mozilla Firefox 132.0

### The OS you're using

GNU+Linux (Gentoo)

### Steps to reproduce

1. Visit https://explore.overturemaps.org/
2. Wait a bit
3. Stare at blank screen
4. ?????
5. File bug report

### The browser URL at the time you encountered the bug

https://explore.overturemaps.org/

**Comment by Bonkles** [comment]
Yes, there very much is supposed to be a map! Yeah this isn't good- no idea what the problem could be though.  Are you able to see the site if you visit from a phone or other device? 

Do you literally just see a blank screen, or do you see any text or map controls at all?  A screenshot or developer tools console log might be helpful.
